Documentation/Signoffs/SteeringSystem.md

```diff
@@ -86,30 +49,46 @@ where C is the number of counts, CPR is the Counts per Revolution, and A is the
 
 Similarly, the accuracy of the angle is highly important to the selection of an encoder. To find the output sensitivy of the encoder, the following equation is used:
      
-     D = 360/(Resolution * 2)       (2)
+     D = 360/(Resolution)       (2)
```

Comment:
The resolution was not analyzed. I saw the Matlab script but it doesn't prove that the resolution is sufficient for your purpose. Probably that is because you may not have a firm idea of what the required resolution is. What resolution is needed for the application?